public/js/Home.js

```diff
@@ -0,0 +1,53 @@
+const requirements = document.getElementById('requirements');
+const textArea = document.getElementById('text-area');
+const skills = document.getElementById('skills');
+const addMore = document.getElementById('add-more');
+const deletBtn = document.getElementById('delete');
+const post = document.getElementById('post');
+const i = 0;
+let j = 0;
+
+function append() {
+  requirements.style.display = 'block';
+}
+addMore.addEventListener('click', () => {
+  j++;
+  const newSkill = document.createElement('input');
+  newSkill.setAttribute('type', 'text');
+  newSkill.setAttribute('placeholder', 'Enter a required skill.');
+  newSkill.setAttribute('id', j);
+  skills.appendChild(newSkill);
+  console.log(newSkill);
+});
+deletBtn.addEventListener('click', () => {
+  if (j >= 1) {
+    j--;
+    skills.removeChild(skills.lastChild);
+  }
+});
+function remove() {
+  if (textArea.value.length === 0) {
+    requirements.style.display = 'none';
+  }
+}
+
+post.addEventListener('click', () => {
+  const skillsArray = [];
+  for (let a = 0; a < j; a++) {
+    skillsArray.push(document.getElementById(a));
```

Comment:
@Anies12 Do you need the value of these elements?  `skillsArray.push(document.getElementById(a).value));`?
Also, another approach here is to iterate over all of the skills inputs like this:
`Array.from( skills.querySelectorAll('input') ).forEach( skillElement => skillsArray.push ....`